### DISCUSSION TITLE
1.1 and 1.2 not in pypi

### DISCUSSION THREAD

**Discussion by jusdino** [latchset/jwcrypto#275] [General]
Hey, super glad you cut tag 1.2 yesterday! Looking now, I see that 1.1 and 1.2 have not made it to pypi yet. Are you intending to publish there?

**Comment by simo5** [comment]
I do not currently have an account on pipy, I was relying on Christian to do it in the past, but he got busy.

**Reply by jusdino** [reply]
Ok. Can I help somehow?

**Comment by simo5** [comment]
I am trying to get maintainership there so I can publish, please be patient

**Comment by simo5** [comment]
Available here now: https://pypi.org/project/jwcrypto/1.2/
